src/Controller/Component/AuthComponent.php

```diff
@@ -349,7 +349,7 @@ protected function _unauthenticated(Controller $controller)
         }
 
         if (!empty($this->_config['ajaxLogin'])) {
-            $controller->viewPath = 'Element';
+            $controller->getview()->viewPath = 'Element';
```

Comment:
Okay, this cleanup is taking lot more time than i anticipated :stuck_out_tongue: